src/content/decentralization/index.md

```diff
@@ -78,33 +165,52 @@ The network works, but it's still a work in progress. Also, like most decentrali
 
 Becoming even more awesome is always on the agenda, though. Here are some things the community is working towards:
 
-- Increased [client diversity](/developers/docs/nodes-and-clients#advantages-of-different-implementations)
-- Better geographical distribution of network operators (active participants in many different countries)
-- Distribution of ETH balances (already one of the most even distributions in crypto)
-- Increased cooperation between the Ethereum community and other decentralized projects (yes, even Bitcoin)
-- Better communication between network operators, users, and software developers
-
-As decentralized network participants, part of our job (should we choose to accept it) is to look for ways we can use and/or support this more equitable structure in our daily lives and our business ventures. Ready to get started?
+- **Increased [client diversity](/developers/docs/nodes-and-clients#advantages-of-different-implementations)**: Diverse software means less chance for a single bug or attack to cause a major problem.
+- **Better geographical distribution of network operators (active participants in many different countries)**: Having nodes in many different places reduces the risk of government interference, outages from regional disasters, etc.
+- **Distribution of ETH balances (already one of the most even distributions in crypto)**: Currency value is at risk for manipulation when the supply is largely controlled by a single entity or group of associated entities. In other words, a centralized group can decide how much is available. Increased distribution reduces this risk.
+- **Increased cooperation between the Ethereum community and other decentralized projects**: Ethereum is at the infrastructure level for the decentralized web. Linking other projects and building more applications increases competition, but it also increases utility.
+- **Better communication between network operators, users, and software developers**: Communication helps identify potential uses and resolve issues. Sometimes one group will have a perspective that the others did not consider.
 
 ## Centralized vs. decentralized {#centralized-vs-decentralized}
```

Comment:
This title should better reflect the content, you're talking about how not all blockchain/crypto projects are decentralized.